src/main/java/projet/projetstage02/threadpool/WorkerThread.java

```diff
@@ -0,0 +1,49 @@
+package projet.projetstage02.threadpool;
+
+public class WorkerThread extends Thread {
```

Comment:
Can you explain to me what this class does

Reply:
its a thread that can be used in the threadpool